src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -978,29 +1024,17 @@ RemainingTestsForCase makeNext(ArrayBuilder<BoundDagTest> remainingTests)
                             // v == K --> !(v == null)
                             trueTestPermitsTrueOther = false;
                             break;
+                        case BoundDagRelationalTest r2:
```

Comment:
>BoundDagRelationalTest [](start = 29, length = 22)

This is the implementation of subsumption and exhaustiveness for relational patterns.